### ISSUE TITLE
Response Undefined in Worksheet Request Error

### ISSUE THREAD

**Issue by lomaxap** [department-of-veterans-affairs/caseflow#10505]
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/3000/

`Cannot ready property 'body' of undefined` 

in [/client/hearings/reducers/index](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/hearings/reducers/index.js#L124)
```javascript
case Constants.HANDLE_WORKSHEET_SERVER_ERROR:
    return update(state, {
      worksheetServerError: {
        // this else condition is needed for 500s
        errors: { $set: action.payload.err.response.body ? action.payload.err.response.body.errors :
          action.payload.err },
        status: { $set: action.payload.err.response.status }
      },
      fetchingWorksheet: { $set: false }
    });
```

`action.payload.err.response` is undefined

**Comment by lomaxap** [comment]
potentially related bug: https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/4850/

**Comment by lomaxap** [comment]
this was incidentally resolved by #10898
